### PR TITLE
Repackaging the OpenPAYGO-Token as a python module

### PR DESCRIPTION
**Pull request by wan5xp**
This can be repackaged into a python module for easy inclusion in other project

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+MANIFEST
+
+# Environments
+.env
+.venv
+env/
+venv/
+ENV/
+env.bak/
+venv.bak/
```
Comment (syenchuk) [inline .gitignore]: Please add a new line at the end of the file

P.S. Personal opinion. I am not a maintainer, just using the library.

File: openpaygo/decode_token.py [hunk 2]
```diff
@@ -12,25 +12,31 @@ def get_activation_value_count_and_type_from_token(cls, token, starting_code, ke
                                                        restricted_digit_set=False, used_counts=None):
         if restricted_digit_set:
             token = OPAYGOShared.convert_from_4_digit_token(token)
-        token_base = OPAYGOShared.get_token_base(token) # We get the base of the token
-        current_code = OPAYGOShared.put_base_in_token(starting_code, token_base) # We put it into the starting code
-        starting_code_base = OPAYGOShared.get_token_base(starting_code) # We get the base of the starting code
-        value = cls._decode_base(starting_code_base, token_base)  # If there is a match we get the value from the token
+        token_base = OPAYGOShared.get_token_base(
+            token)  # We get the base of the token
```
Comment (syenchuk) [inline openpaygo/decode_token.py:16]: The source code doesn't enforce a 80 characters limit… maybe this is not necessary.

**Review by syenchuk — CHANGES_REQUESTED** [review]
Hey, I'm sorry but you reformated everything, so I cannot see the changes.

Could you please submit only changes relevant to your feature?

And open another PR to fix the formatting issue? (however I'm not sure @benmod will accept, many projects have different style guidelines, Django for instance [accepts up to 119 characters per line](https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/coding-style/#python-style))

**Comment by benmod** [comment]
Hi all! Thanks a lot for your work. Sadly I do not have the ability to approve PRs on this repo anymore but I integrated some of your changes into [our Fork](https://github.com/openpaygo/OpenPAYGO-Token) and published it on PyPi here: https://pypi.org/project/openpaygo-token/

You can just do ```pip install openpaygo-token``` to install it.

**Comment by BarnierVi** [comment]
Hi @benmod, sorry there was a problem, now fixed. You shall now be able to approve PRs.

**Comment by Shawnmich7700** [comment]
Please which hardware and version is suitable for the Openpaygo Integration...I have the Arduino UNO R3, is it suitable or is there another board smaller than this for the integration? A picture of the setup will be appreciated.

**Comment by wan5xp** [comment]
There is example implementation for Arduino shown here
https://github.com/EnAccess/OpenPAYGO-HW

Shawnmich7700 ***@***.***> schrieb am Mi. 5. Okt. 2022 um
08:27:

**Comment by Shawnmich7700** [comment]
Thanks for your response...i will proceed with it and revert back to you.
In addition, how can I register on the application or how can I generate
tokens for my customers after programming weekly, and monthly subscriptions?

**Comment by dmohns** [comment]
Resolved via https://github.com/EnAccess/OpenPAYGO-Token/pull/18

**Pull request closed without merging**